testing-endpoints.md

```diff
@@ -18,20 +18,43 @@ curl -X GET http://localhost:8080/api/test/did-debug
 ```
 
 
+#### `api/test/generate-did`
+
+Returns a Long-Form did made from input public keys in jwk format
+
+The body is a JSON object (note, this API requires a content-type header) with the following fields:
+1. `recoveryPublicJwk` - **required** `ES256` public signing key jwk for did recovery operations
+2. `updatePublicJwk` - **required** `ES256` public signing key jwk for did update operations
+3. `signingPublicJwk` - **required** `ES256` public signing key jwk for verifying assertions 
+4. `encryptionPublicJwk` - **optional** public encryption key jwk using the `"alg": "ECDH-ES"` and `"enc": "A256GCM"` encryption algorithm
+5. `domains` - **optional** array of string URI hostnames. Will be added to the did's `LinkedDomains` by an `add-services` action. 
+
+**Note**: The `encryptionPublicJwk` field is optional since 
+not all entities represented by DIDs may need to encrypt documents.
```

Comment:
This key is technically used to *received* encrypted documents; I'm happy leaving it optional here though.